src/Docker.DotNet/Models/ContainerExecInspectResponse.Generated.cs

```diff
@@ -5,7 +5,7 @@ namespace Docker.DotNet.Models
     [DataContract]
     public class ContainerExecInspectResponse // (types.ContainerExecInspect)
     {
-        [DataMember(Name = "ExecID", EmitDefaultValue = false)]
+        [DataMember(Name = "ID", EmitDefaultValue = false)]
```

Comment:
Did we always have this wrong of did Docker change this?

Reply:
It looks like this was updated February 2020
https://github.com/moby/moby/commit/cc993a9cbf40c1a94b5bada84f1d759b72df5c4e

Reply:
Nice thanks.